### PR TITLE
Apache Kafka channel install

### PR DESCRIPTION
**Pull request by matzew**
based on changes from #2193 - let's get that in first!

/assign @mattmoor 

A couple of thoughts....  

Should we separate this into three different aspects? 

* Broker enablement (well, atm there is just one, but we could show the install of a custom one, w/o the labeling?
* Full dedicated section on Sources _and_ one that covers channels.

Currently we mix all in one, but as soon we have more content there, it might be better to separate things!  Happy to help on that.

### CODE REVIEW

File: docs/install/knative-with-any-k8s.md [hunk 1]
```diff
@@ -444,25 +444,41 @@ To learn more about the Github source, try [our sample](../eventing/samples/gith
...
    ```bash
    kubectl apply --filename https://github.com/knative/eventing-contrib/releases/download/{{< version >}}/kafka-source.yaml
    ```
 
-To learn more about the Kafka source, try [our sample](../eventing/samples/kafka/source/README.md)
+To learn more about the Apache Kafka source, try [our sample](../eventing/samples/kafka/source/README.md)
+
+{{< /tab >}}
+
+{{% tab name="Apache Kafka Channel" %}}
```
Comment (mattmoor) [inline docs/install/knative-with-any-k8s.md]: This is the source section, can we move this to the set of Channel tabs above?  there is a TODO

File: docs/install/knative-with-any-k8s.md [hunk 2]
```diff
@@ -444,25 +444,41 @@ To learn more about the Github source, try [our sample](../eventing/samples/gith
...
+The following command installs the Apache Kafka Channel:
+
+   ```bash
+   curl -L "https://github.com/knative/eventing-contrib/releases/download/{{< version >}}/kafka-channel.yaml" \
+    | sed 's/REPLACE_WITH_CLUSTER_URL/my-cluster-kafka-bootstrap.kafka:9092/' \
+    | kubectl apply --filename -
+   ```
+
+Before you can install the Apache Kafka channel, you need to have a running installation of Apache Kafka.
```
Comment (mattmoor) [inline docs/install/knative-with-any-k8s.md]: If you have to do this first, then you should say this first, folks tend not to read thru before doing.

TAL at the Serving Istio instructions for an example of this.  That said, I'd prefer if we could inline a simple guide (Istio has an issue for this).

File: docs/install/knative-with-any-k8s.md (hunk 1 above)
Comment (matzew) [reply docs/install/knative-with-any-k8s.md]: ah - that answers some of the thoughts I have in the text/desc. of my PR

**Comment by mattmoor** [comment]
/lgtm
/assign @samodell 

This LGTM, kicking over to Sam for approval.

File: docs/install/knative-with-any-k8s.md [hunk 3]
```diff
@@ -364,8 +364,23 @@ The following command installs an implementation of Channel that runs in-memory.
 
 {{< /tab >}}
 
+{{% tab name="Apache Kafka Channel" %}}
+
+1. [Installing Apache Kafka for Kubernetes](../eventing/samples/kafka/README.md)
```
Comment (samodell) [inline docs/install/knative-with-any-k8s.md]:
```suggestion
1. First, [Install Apache Kafka for Kubernetes](../eventing/samples/kafka/README.md)
```

File: docs/install/knative-with-any-k8s.md [hunk 4]
```diff
@@ -364,8 +364,23 @@ The following command installs an implementation of Channel that runs in-memory.
...
+
+1. Install the Apache Kafka Channel:
```
Comment (samodell) [inline docs/install/knative-with-any-k8s.md]:
```suggestion
1. Then install the Apache Kafka Channel:
```

**Comment by matzew** [comment]
@samodell updated! thx for review!

**Comment by samodell** [comment]
/lgtm
/approve

No problem! Thanks for the PR :)

**Pull request merged**